tests/unit/utils/test_txn_parser.py

```diff
@@ -0,0 +1,476 @@
+import copy
+from unittest import TestCase
+
+import xrpl.utils
+
+TEST_TXN = {
+    "engine_result": "tesSUCCESS",
+    "engine_result_code": 0,
+    "engine_result_message": """
```

Comment:
Why does this have a `.strip()` instead of having the string just be there as is?